src/Core/src/Platform/Android/ContextExtensions.cs

```diff
@@ -121,6 +121,9 @@ internal static int GetDisabledThemeAttrColor(this Context context, int attr)
 			}
 		}
 
+		public static bool TryResolveAttribute(this Context context, int id, out float? value) =>
```

Comment:
Why is this needed?

Reply:
for this
https://github.com/dotnet/maui/pull/1774/files/1583bbb10df6a0b72581d12b008fbab4864b34e3#diff-ce2735a0580c21a5e576b81a84822093bf8bee64175e0d12700c5c50dc99ed44R682